examples/simple-game-server/main.go

```diff
@@ -313,7 +313,14 @@ func handleResponse(txt string, s *sdk.SDK, cancel context.CancelFunc) (response
 			responseError = fmt.Errorf("Invalid GET_COUNTER_COUNT, should have 1 arguments")
 			return
 		}
-		response, responseError = getCounterCount(s, parts[1])
+		count, err := getCounterCount(s, parts[1])
+		if err != nil {
```

Comment:
In general I'd recommend keeping formatting changes within the getCounterCount method.

Reply:
Updated the format changes in getCounterCount() and other methods.